### PR TITLE
chore(swap): include boost event data in swap status

### PR DESCRIPTION
**Pull request by GabrielBuragev**

### CODE REVIEW

File: packages/swap/src/routes/swap.ts [hunk 1]
```diff
@@ -295,6 +295,8 @@ router.get(
       depositChannelAffiliateBrokers: affiliateBrokers,
       depositChannelMaxBoostFeeBps: channel?.maxBoostFeeBps,
       effectiveBoostFeeBps,
+      swapBoostedAt: swap?.swapBoostedAt?.valueOf(),
```
Comment (niklasnatter) [inline packages/swap/src/routes/swap.ts:298]: maybe `depositBoostedAt` would be more correct?

Comment (niklasnatter) [reply packages/swap/src/routes/swap.ts:298]: now that i think of it, wouldnt this be the same as `depositReceivedAt`

Comment (niklasnatter) [reply packages/swap/src/routes/swap.ts:298]: and should we have a `boostSkippedAt` too?

Comment (GabrielBuragev) [reply packages/swap/src/routes/swap.ts:298]: > now that i think of it, wouldnt this be the same as `depositReceivedAt`

ahh yeah, now i remember why i didn't include it int he first place 🙈

File: packages/swap/src/routes/swap.ts [hunk 2]
```diff
@@ -295,6 +295,10 @@ router.get(
       depositChannelAffiliateBrokers: affiliateBrokers,
       depositChannelMaxBoostFeeBps: channel?.maxBoostFeeBps,
       effectiveBoostFeeBps,
+      depositBoostedAt: swap?.depositBoostedAt?.valueOf(),
```
[the remark this answers is not in the document]
Comment (niklasnatter) [reply packages/swap/src/routes/swap.ts:298]: also would be nice to add a test case that has the properties

**Review by niklasnatter — APPROVED** [review]

File: packages/swap/src/routes/swap.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (GabrielBuragev) [reply packages/swap/src/routes/swap.ts:298]: Done

File: packages/sdk/src/swap/types.ts [hunk 3]
```diff
@@ -82,6 +82,8 @@ interface DepositAddressFields extends SwapStatusResponseCommonFields {
   depositChannelAffiliateBrokers?: { account: string; commissionBps: number }[];
   depositChannelMaxBoostFeeBps: number;
   effectiveBoostFeeBps?: number;
+  boostSkippedAt?: number;
+  boostKippedBlockIndex?: string;
```
Comment (jerryafr) [inline packages/sdk/src/swap/types.ts:86]:
```suggestion
  boostSkippedBlockIndex?: string;
```

**Pull request merged**